core/src/main/kotlin/xtdb/util/TemporalBounds.kt

```diff
@@ -20,11 +21,19 @@ class TemporalBounds {
         fun gte(operand: Long) = apply { lower = max(operand, lower) }
 
         fun inRange(operand: Long) = operand in lower..upper
+
+        override fun toString(): String {
```

Comment:
these might be better achieved by making TemporalBounds and TemporalColumn data classes, if that works?

Reply:
Can make them data classes. The override for `TemporalColumn` still needs to happen if I want a human friendlier output or is there a way to annotate data fields with special printers.